docs/mobile-apps/features/audio-capture.md

```diff
@@ -4,7 +4,9 @@ title: Audio Capture
 ---
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<p><small><span className="sauceDBlue">Real Devices Only</span></small></p>
+<p><small><span className="sauceDBlue">
```

Comment:
The badge now says "Devices Only" -- Is that intended?

Reply:
it is not!